cypress/integration/seededFlows/articleFlows/commentOnArticle.spec.js

```diff
@@ -631,4 +631,54 @@ describe('Comment on articles', () => {
     cy.findByRole('button', { name: /^Toggle dropdown menu$/i }).click();
     cy.findByRole('link', { name: /^Edit this comment$/i }).should('not.exist');
   });
+
+  it('should toggle aria-pressed button', () => {
+    cy.visitAndWaitForUserSideEffects('/admin_mcadmin/test-article-slug');
+
+    // SVG should exists with Like comment: as title
+    cy.findByRole('img', { name: 'Like comment:' });
+
+    cy.findByTestId('comments-container').within(() => {
+      cy.findByRole('button', { name: /^heart$/i })
+        .as('likeButton')
+        .should('exist')
+        .and('have.attr', 'aria-pressed', 'false')
+        .and('not.have.class', 'reacted');
+
+      cy.get('@likeButton').within(() => {
+        cy.get('span.reactions-count').should('have.text', '0');
+        cy.get('span.reactions-label').should(($span) => {
+          expect($span.text().trim()).equal('Like');
+        });
+      });
```

Comment:
It's interesting to me that Cypress interprets the accessible name of the button as "heart", when it should be "Like comment: 0 likes" (as we intended). I had a quick look and it seems this is a known issue with the API Testing Library uses to check the accessible name, in that it takes the `title` over everything else 🤷‍♀️ 

I've double-checked in Safari/Chrome/Firefox and the correct accessible name is surfaced, so I think we're all good, and nice work working around this in the Cypress tests.